covid19-etl/etl/cityofchicago.py

```diff
@@ -0,0 +1,410 @@
+#  This ETL is for city of chicago dataset for COVID-19 Daily Cases, Deaths, and Hospitalizations (CDH)
+#  Reference: https://data.cityofchicago.org/Health-Human-Services/COVID-19-Daily-Cases-Deaths-and-Hospitalizations/naz8-j4nc
+
+import boto3
+import csv
+from datetime import datetime
+import time
+import json
+import requests
+import pandas as pd
+from contextlib import closing
+from utils.metadata_helper import MetadataHelper
+from utils.format_helper import (
+    derived_submitter_id,
+    format_submitter_id,
+)
+from etl import base
+import sys
+
+CITYOFCHICAGO_CDH_URL = "https://data.cityofchicago.org/resource/naz8-j4nc.csv"
+
+
+class CITYOFCHICAGO(base.BaseETL):
```

Comment:
i think a unit test with a fake input CSV file, that initializes a `CITYOFCHICAGO` instance, then calls `files_to_submissions()`, and then checks the contents of `self.summary_locations`, `self.summary_clinicals` and `self.summary_group_demographics` would be very good. Is that something you would have time to add?

Reply:
I am not sure where and how to add the test. Can you give me some instructions and example for that?

Reply:
the unit tests are here https://github.com/uc-cdis/covid19-tools/tree/master/covid19-etl/tests
give me a few, i'll push something for you to get started

Reply:
Sure. Thank you :) I should be creating a test case which should include dummy data submission (adding data into `summary_location`, `summary_clinical` and `summary_group_demographics`) and retrieval of it from guppy and verify?

Reply:
@urvi-occ you can cherry-pick this commit: `594739a9f8935babe1ed8e7a9ebb49bcce1f90ae` (https://github.com/uc-cdis/covid19-tools/commit/594739a9f8935babe1ed8e7a9ebb49bcce1f90ae)

all you have to do is update the assertions with the expected data

and add tests for edge cases if you can think of any

Reply:
That make sense. Thank you. Is it okay if I directly contribute to `test/cityofchicago` branch or should I make copy of that and make a PR to `test/cityofchicago`  and then ask for a merge to master?

Reply:
`git cherry-pick 594739a9f8935babe1ed8e7a9ebb49bcce1f90ae`

Reply:
oh and the name of the test should be updated, i copy/pasted and forgot to change it

Reply:
For sure, I would be working on that soon

Reply:
@paulineribeyre 

> oh and the name of the test should be updated, i copy/pasted and forgot to change it

Can you help me locate this 😅

Reply:
the test name here: https://github.com/uc-cdis/covid19-tools/blob/8c8a8b300b1668d84ce5dae1f286f75e6f11fed7/covid19-etl/tests/test_cityofchicago.py#L39

Reply:
Hey, I have added test cases and some changes in the input file to check with the missing values. Let me know if the test cases I added are okay or not